### PR TITLE
CI: fix build

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Missing stuff from both latest PR's

### CODE REVIEW

**Comment by sdroege** [comment]
How did that pass the CI in the other PR?

**Review by sdroege — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
There was no CI to run in the previous PR. :)

File: src/auto/x11_visual.rs [hunk 1]
```diff
@@ -16,15 +15,7 @@ glib_wrapper! {
...
-impl X11Visual {
-    pub fn get_xvisual(&self) -> Option<xlib::Visual> {
-        unsafe {
-            from_glib_none(gdk_x11_sys::gdk_x11_visual_get_xvisual(
-                self.to_glib_none().0,
-            ))
-        }
-    }
-}
+impl X11Visual {}
```
Comment (GuillaumeGomez) [inline src/auto/x11_visual.rs:18]: Oh, I guess that's a bug in `gir`. :)

**Comment by bilelmoussaoui** [comment]
The ci is failing currently due to this missing feature https://github.com/gtk-rs/sys/pull/186

**Comment by bilelmoussaoui** [comment]
It's all green now :) needs some review regarding the feature removal from rt.rs

**Comment by GuillaumeGomez** [comment]
It seems correct to me. Anything invalid for you @sdroege?

File: src/auto/x11_visual.rs (hunk 1 above)
Comment (sdroege) [reply src/auto/x11_visual.rs:18]: https://github.com/gtk-rs/gir/issues/976

**Review by sdroege — APPROVED** [review]
Looks good to me. But someone should probably look at the clippy warnings at some point :)

For the unused imports I went over to

```rust
#[allow(unused_imports)]
mod auto;
```

in GStreamer now.

**Comment by bilelmoussaoui** [comment]
> Looks good to me. But someone should probably look at the clippy warnings at some point :)
> 
> For the unused imports I went over to
> 
> ```rust
> #[allow(unused_imports)]
> mod auto;
> ```
> 
> in GStreamer now.

Yeah I will add more tests in the future :)

**Pull request merged**